docs/src/api/class-request.md

```diff
@@ -85,6 +85,15 @@ Name of the header.
 
 Whether this request is driving frame's navigation.
 
+## method: Request.isServiceWorkerRequest
```

Comment:
Are we sure we want this one? It is equivalent to `!!request.serviceWorker()` after all.

Reply:
Agreed—removed.